src/GitHubExtensionServer/Program.cs

```diff
@@ -140,6 +152,7 @@ private static void LogPackageInformation()
               "MicrosoftWindows.Client.WebExperience_cw5n1h2txyewy",
               "Microsoft.Windows.DevHome_8wekyb3d8bbwe",
               "Microsoft.Windows.DevHomeGitHubExtension_8wekyb3d8bbwe",
+              "Microsoft.Windows.DevHomeGitHubExtension.Dev_8wekyb3d8bbwe",

```

Comment:
Should this not be included, or should we include canary as well?

Reply:
We should include all of the devhome and extension names so the logging of packages present is complete.